src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -39,10 +39,14 @@ public virtual void GatedPublishRunner(Permutation permutation)
 
         void Tasks(Permutation permutation, [CallerMemberName] string memberName = "")
         {
-            permutation.Category = GetType().Name;
+            var fixture = GetType().GetCustomAttribute<TestFixtureAttribute>();
+            permutation.Category = fixture.Category;
+            permutation.Description = fixture.Description;
```

Comment:
What is the benefit of passing a description all the way to the NLog global context and add it to each and every line?